econet_hvac_odu.yaml

```diff
@@ -252,6 +252,22 @@ sensor:
     request_mod: none
     accuracy_decimals: 0
     internal: true
+  - platform: econet
+    name: "ODU Size (Tons)"
+    id: odu_size
+    sensor_datapoint: ODU_SIZE
+    request_once: true
+    request_mod: 7
```

Comment:
I'm not sure if this should share a `request_mod` with other ODU sensors vs. going in a `request_mod` with other `request_once` sensors. Might be a question for @tronikos?

Reply:
It's fine sharing request_mod as long as the MCU responds. It's mostly necessarily for large objects such alerts that have long strings.